libensemble/__init__.py

```diff
@@ -4,7 +4,10 @@
 Library to coordinate the concurrent evaluation of dynamic ensembles of calculations.
 """
 
-__version__ = "0.7.1+dev"
+with open("VERSION", "r") as f:
```

Comment:
@shuds13 This is being invoked where the libE calling script is, not where `__init__.py` resides. We don't want to require a `VERSION` file where the calling script is. Is there a way to refer to the base directory of the package? Something like
`with open(basedir + "libensemble/VERSION", "r") as f:`
?

Reply:
Yes, just thinking what is the best method